lib/cylc/gui/gsummary.py

```diff
@@ -537,9 +541,9 @@ def _on_query_tooltip(self, widget, x, y, kbd_ctx, tooltip):
             host = model.get_value(parent_iter, 0)
             suite = model.get_value(parent_iter, 1)
             child_row_number = path[-1]
-        update_time = model.get_value(iter_, 4)
+        suite_update_times = model.get_value(iter_, 4)
```

Comment:
Sorry, I meant `....times` just for the dictionary type variables in the `update` method and others - this one
is better as `....time`.